common/code/ArisBasics.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <cstdint>
+
+enum class SonarFrequency : uint32_t { Low = 0, High = 1 };
```

Comment:
Will add a note that these serve the purpose of preventing ingress of the protoc generated types into our code base.